README.md

```diff
@@ -1,22 +1,32 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+  # Capstone Module 1: Hemp Industries Submmit 2021
 
-> One paragraph statement about the project.
+> This is the final project regading the firt module of Microverse program. 
+> The main subjects that we intenselly worked on, were the three pilars of any 
+> website on mobile or desktop devices, wich are HTML,CSS & JavaScript. I have to confess that, 
+> even when I spare lots of hours reading the basics and some advanced features, It was 
+> one of the biggest intellectual challengues I've been trough in all my live. I will enhance
+> you to keep reading this document and find what I personally exprienced over the past 5 weeks
+> of intense learning, lots of sleep depravation, frustrations and satisfactions over this period 
+> of time. 
```

Comment:
- [ ] KIndly the description of the project should not be your experience at microverse the last 5 weeks rather should be a short summary of what the project does. I think it would be better if you change it with an appropriate one.

Reply:
I had a good laugh. You are right, this isn´t about me.